### PR TITLE
fix: Remove FenwickTree

### PR DESCRIPTION
**Pull request by matgrz1993**
FenwickTree is the same Data Structure as Binary Indexed Tree located in file "range_queries/bit.cpp" so it could be removed.

#### Checklist

- [x] Added description of change
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTION.md#Commit-Guidelines)
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

**Comment by kvedala** [comment]
Good catch, @matgrz1993  I agree. The other program you are referring to: `range_queries/bit.cpp` is more refined albeit without documentation. 
Can you also change the ubiquitous name of `bit.cpp` to `fenwick_tree.cpp`? Once you make the commit, there maybe errors from `cpplint` because of the updated repo standards since the time of the original code. If you could add documentation to the code, that would be just wonderful We are following the [doxygen](https://www.doxygen.nl) standard ([reference code](https://github.com/kvedala/C-Plus-Plus/blob/master/math/realtime_stats.cpp))

Thank you.

**Comment by matgrz1993** [comment]
It's not my catch :)
It's already reported bug:
https://github.com/TheAlgorithms/C-Plus-Plus/issues/811

File: range_queries/fenwick_tree.cpp [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ *
+ * A Fenwick tree or binary indexed tree is a data structure 
+ * that can efficiently update elements and calculate 
+ * prefix sums in a table of numbers.
+ */
+#include <cassert>
+#include <iostream>
+#include <vector>
+
+class FenwickTree {
```
Comment (kvedala) [inline range_queries/fenwick_tree.cpp:13]: Could you document the classes and members as shown in there cue above

Comment (matgrz1993) [reply range_queries/fenwick_tree.cpp:17]: I documented some. Please take a look.

**Comment by kvedala** [comment]
@matgrz1993 could you please resolve on the merge by merging the main repo master branch? Thank you

**Comment by matgrz1993** [comment]
@kvedala Done.

**Comment by kvedala** [comment]
@ayaankhan98 can be merged

**Comment by Panquesito7** [comment]
This program seems not to work properly.
I did some quick tests on GitPod and this is the output:

GitPod tests

There are also various functions with missing documentation.
Test implementations should be made in a separate `tests` function.

**Pull request merged**